app/src/main/java/com/keylesspalace/tusky/db/Converters.kt

```diff
@@ -61,13 +63,13 @@ class Converters {
         return str?.split(";")
                 ?.map {
                     val data = it.split(":")
-                    createTabDataFromId(data[0], data.drop(1))
+                    createTabDataFromId(data[0], data.drop(1).map { s -> URLDecoder.decode(s, "UTF-8") })
                 }
     }
 
     @TypeConverter
     fun tabDataToString(tabData: List<TabData>?): String? {
-        return tabData?.joinToString(";") { it.id + ":" + it.arguments.joinToString(":") }
+        return tabData?.joinToString(";") { it.id + ":" + it.arguments.joinToString(":") { s -> URLEncoder.encode(s, "UTF-8") } }
```

Comment:
List name may includes ":".

Reply:
put that into an comment in the code please